### PR TITLE
feat(deposit): show success message in case of manual review of deposit action

### PR DESCRIPTION
**Pull request by milan-bc**
## Description (optional)
Add a concise explanation of the changes.
manual review

## Testing Steps (optional)
Detail the steps required for the reviewer(s) to verify and test these changes.

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts [hunk 1]
```diff
@@ -344,7 +344,12 @@ export default ({ api, coreSagas, networks }: { api: APIType; coreSagas: any; ne
   const ClearedStatusCheck = function* (orderId) {
     const order: BSTransactionType = yield call(api.getPaymentById, orderId)
 
-    if (order.state === 'CLEARED' || order.state === 'COMPLETE' || order.state === 'FAILED') {
+    if (
+      order.state === 'CLEARED' ||
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts:348]: Why not use an enum for this states?

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/DepositStatus/index.tsx [hunk 2]
```diff
@@ -34,7 +34,8 @@ const DepositStatus = (props) => {
     Success: (val) =>
       props.formValues?.order?.state === 'CLEARED' ||
       props.formValues?.order?.state === 'COMPLETE' ||
-      props.formValues?.order?.state === 'COMPLETED' ? (
+      props.formValues?.order?.state === 'COMPLETED' ||
+      props.formValues?.order?.state === 'MANUAL_REVIEW' ? (
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/DepositStatus/index.tsx:38]: and prevent hard code strings duplication

File: packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts (hunk 1 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/components/brokerage/sagas.ts:348]: this was a quick fix, but yeah I do agree. I personally force to use enums all around app :)

File: packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/DepositStatus/index.tsx (hunk 2 above)
Comment (milan-bc) [reply packages/blockchain-wallet-v4-frontend/src/modals/Brokerage/Banks/Deposit/DepositStatus/index.tsx:38]: actually there are 2 diff COMPLETE and COMPLETED :)

**Comment by blockdylanb** [comment]
Need to hide the "funds are spendable immediately" text for manual review. I can do it.

File: packages/blockchain-wallet-v4/src/network/api/buySell/types.ts [hunk 3]
```diff
@@ -327,20 +335,24 @@ export type BSTransactionsType = {
...
+  FRAUD_REVIEW = 'FRAUD_REVIEW',
+  MANUAL_REVIEW = 'MANUAL_REVIEW',
+  PENDING = 'PENDING',
+  PENDING_DEPOSIT = 'PENDING_DEPOSIT',
+  REFUNDED = 'REFUNDED',
+  REJECTED = 'REJECTED',
+  UNIDENTIFIED = 'UNIDENTIFIED'
+}
+
+export type BSTransactionStateType = keyof typeof BSTransactionStateEnum
```
Comment (julianromero-bc) [inline packages/blockchain-wallet-v4/src/network/api/buySell/types.ts:355]: 🤩

**Review by julianromero-bc — APPROVED** [review]

**Pull request merged**